server/src/main/java/io/druid/client/ImmutableDruidDataSource.java

```diff
@@ -72,4 +72,14 @@ public boolean isEmpty()
   {
     return segmentsHolder;
   }
+
+  @Override
+  public String toString()
+  {
+    return "ImmutableDruidDataSource{"
+           + "name='" + name
+           + "', segments='" + segmentsHolder
```

Comment:
Hmm, actually I followed some other toString() implementaions like ```DataSegment``` or ```SegmentDescriptor```. It seems we need to make a standard for toString() first.